cmdx.py

```diff
@@ -511,7 +511,7 @@ class Node(object):
         >>> transform = createNode("transform")
         >>> transform["tx"] = 5
         >>> transform["worldMatrix"][0] >> decompose["inputMatrix"]
-        >>> decompose["outputTranslate"]
+        >>> decompose["outputTranslate"].read()
```

Comment:
Hmm, having to put `.read()` here is a bummer. I can't think of any scenario where this affects code outside of these doctests, but at least we should include a few with the result without `.read()` so we can see what they look like. Can you include a few?

Reply:
Another option would be to revert these and just change the output to accommodate the new repr result. Did you still want to have both?

Reply:
A mixture would be best, we both want to test these outputs and also document them.